server/api_server/mcp_endpoint.py

```
@@ -1,379 +1,975 @@
+
+
+# =============================================================================
+# MCP RESOURCES
+# =============================================================================
+
+def _list_resources() -> List[Dict[str, Any]]:
+    """List available MCP resources (read-only data like logs)."""
+    resources = []
+
+    # Log files
+    log_dir = os.getenv("NETALERTX_LOG", "/tmp/log")
```

Ruff (0.14.11)

Probable insecure usage of temporary file or directory: "/tmp/log"

(S108)